configs/setups/awicm3/awicm3.yaml

```diff
@@ -105,6 +105,7 @@ general:
 
         # Load compute recipe from oifs.yaml
         prepcompute_recipe: ${oifs.prepcompute_recipe}
+        tidy_recipe: ${fesom.tidy_recipe}
```

Comment:
Wouldn't it be possible to integrate the changes of a recipe here in a choose block? So that the changed recipe is actually only taken if necessary?
In other words: If I don't want to use the plugin `tar_binary_restarts` that will be called in the changed recipe, I don't need a changed recipe here.
As I understand it now, everybody using AWICM3 needs/will install this plugin because of a changed recipe, but may never use it. Or do I have a thinking error here?

Reply:
>  As I understand it now, everybody using AWICM3 needs/will install this plugin because of a changed recipe, but may never use it. Or do I have a thinking error here?

With these lines
https://github.com/esm-tools/esm_tools/pull/1075/files#diff-455b0ceb6e69ed7e3a2b286473f7b9b0a0c99a59892b0d5feee4f87f6d60d53cR12-R14
you are saying "everyone that installs AWICM3 will also install this plugin"

That should remain, cause you don't have a way to tell esm_master that you need or not tar bin plugin.

This is another way of doing things, you either put the switch within the yamls, or in the plugin. It's the same essentially.
```diff
-tar_bin = fesom.tar_binary_restart
-choose_tar_bin:
+choose_tar_binary_restarts:
    "true":
        tidy_recipe: ${fesom.tidy_recipe}
```

Reply:
That being said, I would leave it as it currently is. It's cleaner and the if is in the plugging itself